java-jboss/README.md

```diff
@@ -0,0 +1,41 @@
+# Quickstart for Java + JBoss debugging
+
+A sample application for debugging Java + JBoss apps using Rookout.
+
+Before following this guide we recommend reading the basic [Java + Rookout] guide.
+
+## Rookout Integration Explained
+
+There are 4 simple steps to integrate Rookout into your existing JBoss web application:
+
+1. Add the following JAVA_OPTS arguments in the standalone config script:
+```bash
+-javaagent:<PATH_TO_ROOK> -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager 
```

Comment:
Why do we need so many flags?

Reply:
We need that to fix the logger issues and class loading failures on startup

Reply:
removing any of these flags will crash the program